games/e.yaml

```diff
@@ -212,7 +214,8 @@
 
 - name: [Elite II, 'Frontier: Elite II']
   meta:
-    genre: [action,simulation]
+    genre: [Action, Simulation]
+    subgenre: [Flight Simulator]
```

Comment:
I think `flight` could become it's own main genre. Maybe something to be created as a new issue after this gets merged? Certainly not a blocker.